README.md

```diff
@@ -54,4 +54,4 @@ Requires a [Go installation](https://go.dev/dl/) to run the examples.
   - Could yield hints when coder requests it.
 
 #### Long term
-* Have exercises that cover the entire [Go spec](https://go.dev/ref/spec).
```

Comment:
Why is git diffing this line? I can't tell them apart

Reply:
i'm not sure; i noticed that as well, but i didn't touch that line at all in my edit